### DISCUSSION TITLE
Conseguir Conquistas do GitHub

### DISCUSSION THREAD

**Discussion by clowngames13** [community/community#133049] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

**Vi alguns perfis com umas conquistas no GitHub, fiquei interessado, queria saber como é possível pega-las se possível!**

**Comment by itvinl** [comment]
**Olá, boa noite!**

_Há um repositório no GitHub que tem todas as informações relacionadas a essas conquistas, procure na barra de pesquisa do seu GitHub "Thinkright20/Profile-Badges", lá estará o repositório com todas as informações que você precisa saber._

**Reply by clowngames13** [reply]
Muito obrigado! Vou conferir
